### DISCUSSION TITLE
Main

### DISCUSSION THREAD

**Discussion by Alexiscomete** [Dibi-programming-langage/Skribi-langage-source#20] [General]
Salut @PiggyPigCute , on peut parler ici

**Comment by PiggyPigCute** [comment]
Flemme, discord c'est plus pratique

**Reply by Alexiscomete** [reply]
En effet, après ici je reçois un mail

**Reply by Alexiscomete** [reply]
Si tu as des trucs sur le code on peut aussi parler ici en tout cas, surtout si un jour on a des contributeurs extérieurs à Discord
